### PR TITLE
feat: Added support for zsh on dev setup

### LINKED ISSUE #3: Setup does not check which shell user is using
When doing `dev setup` this message shows up:
`grep: /Users/username/.bash_profile: No such file or directory`

Might want to check which shell user is using and add the "dev" alias to correct file.

### PR DESCRIPTION
**Pull request by RostiMelk**

### CODE REVIEW

**Comment by RostiMelk** [comment]
@torbratsberg Can you test setup?

File: functions/setup.sh [hunk 1]
```diff
@@ -8,9 +8,23 @@ setup() {
     ${warning}Web Devkit setup wizard${end}${NL}You have to complete this setup wizard in order to use site cloner."
     
     # Adds an alias
-    bash_profile_path="/Users/$USER/.bash_profile"
-    if ! grep -qF "alias $prefix" $bash_profile_path ; then
-        printf "${NL}alias $prefix=\"bash $scriptpath/start.sh\"" >> $bash_profile_path
+    case $0 in
```
Comment (torbratsberg) [inline functions/setup.sh]: This $0 seems to be the full path to the start.sh file. Could $SHELL be used in stead?

Comment (RostiMelk) [reply functions/setup.sh]: @torbratsberg New commit added!

**Comment by RostiMelk** [comment]
@torbratsberg Good to go?

**Pull request merged**